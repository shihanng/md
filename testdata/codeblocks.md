Inline `code1 code2 code3\n`

    indent 4 spaces

```
Or use 3 backticks
```

```go
var num = 42
```
