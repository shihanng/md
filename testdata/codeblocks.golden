Inline code

```
indent 4 spaces
```

```
Or use 3 backticks
```

```go
var num = 42
```
